### PR TITLE
improvement(dashboard): change menu layout and update styling

### PR DESCRIPTION
**Pull request by edvald**
This was overdue, a bit of a styling update across the (dash)board.

### CODE REVIEW

**Comment by eysi09** [comment]
Could you share some screenshots with the changes you made?

**Comment by eysi09** [comment]
The modules on the overview page seemed to be vertically aligned to the middle. Both on my by laptop screen and external monitor. This is only visible if the module only occupy a single row, if they wrap, it looks normal. I'm on latest Chrome.

Screenshot 2021-03-08 at 10 43 49

**Comment by edvald** [comment]
Hmm, right. Any ideas on what may be causing that? I'm very confused by the different types of layouts being used concurrently there.

File: dashboard/src/components/menu.tsx [hunk 1]
```diff
@@ -9,104 +9,106 @@
...
   } = useUiState()
 
   return (
     <>
-      <SidebarToggleButton onClick={toggleSidebar}>
-        {isSidebarOpen ? <CloseSidebarIcon /> : <OpenSidebarIcon />}
-      </SidebarToggleButton>
-      <SidebarContainer visible={isSidebarOpen}>
-        <div className={"ml-1"}>
+      {/* <MenuToggleButton onClick={toggleMenu}>
```
[the remark this answers is not in the document]
Comment (edvald) [reply dashboard/src/components/menu.tsx:90]: I wasn't sure if we'd want to revive the toggling at some point. If we feel sure we won't, then I can remove all that.

**Comment by edvald** [comment]
Should be all set now, cleaned up a bit and fixed that styling issue.

**Review by eysi09 — APPROVED** [review]

**Pull request merged**